build-parent/pom.xml

```diff
@@ -81,13 +81,13 @@
         <junit-pioneer.version>2.2.0</junit-pioneer.version>
 
         <!-- Database images for JDBC/Reactive/Hibernate tests and devservices -->
-        <postgres.image>docker.io/postgres:14</postgres.image>
+        <postgres.image>docker.io/postgres:16</postgres.image>
         <mariadb.image>docker.io/mariadb:10.11</mariadb.image>
         <db2.image>icr.io/db2_community/db2:11.5.9.0</db2.image>
         <mssql.image>mcr.microsoft.com/mssql/server:2022-latest</mssql.image>
-        <mysql.image>docker.io/mysql:8.0</mysql.image>
+        <mysql.image>docker.io/mysql:8.0.33</mysql.image>
```

Comment:
LGTM, though I'm surprised this is even needed. I'd have expected `8.0` to point to the latest `8.0.x`.

Reply:
If the container producer sync the N.N version with N.N.x the N.N should be enough and it should always check if the image repository have changes.

Looking at MySQL and mongo and see that they have N.N version available

Reply:
Bumped to 8.4 LTS